ResourceManager/GitExtensionsControl.cs

```diff
@@ -9,10 +9,76 @@
 
 namespace ResourceManager
 {
+    public class ThemedUserControl : UserControl
```

Comment:
Is there any reason not to enhance `GitExtensionsControl` directly?

Reply:
There is no right answer.
I know the ThemedUserControl control consolidate visuals, but I don't know what the GitExtensionsControl control is responsible for.

Needs yours proposition.